test/local/account_routes.js

```diff
@@ -673,3 +675,110 @@ test(
     )
   }
 )
+
+test(
+  '/account/create validates metrics context data',
```

Comment:
@philbooth yes, I think you're right, a lot of the setup code here could be shared.  I like the way it's done in content-server with multiple levels of nesting, each a `beforeEach` and `afterEach` to manage shared setup and teardown code.  Perhaps we can construct something similar for this file?